src/rviz/properties/property.h

```diff
@@ -136,6 +136,40 @@ class RVIZ_EXPORT Property : public QObject
            const char* changed_slot = nullptr,
            QObject* receiver = nullptr);
 
+  template <class Functor>
```

Comment:
I would like to replace this generic functor with the specific function type required here. I made a corresponding commit for illustration.

Reply:
I'm not sure this is possible for all properties. Some slots can have extra function arguments associated with them (might be wrong on this though)

Reply:
The signature of the `changed()` signal is fixed as far as I remember.

Reply:
Yeah, I'll pick this up soon. Had a change of computers (and OS; discovered I have an allergy to the new OS)

Reply:
PTAL. I've modified all the "new" code to not use templates